packages/core/src/modal/modal.element.ts

```diff
@@ -55,13 +49,11 @@ import { styles } from './modal.element.css.js';
  * @cssprop --content-box-shadow-color
  * @cssprop --width
  */
-export class CdsModal extends CdsBaseFocusTrap {
+export class CdsModal extends CdsInternalOverlay {
   static get styles() {
-    return [baseStyles, styles];
+    return [baseStyles, overlayStyles, styles];
```

Comment:
You can inherit base component styles without needing to import with this syntax

```typescript
  static get styles() {
    return [...super.styles, styles];
  }
```